smart_proxy_salt.gemspec

```diff
@@ -13,4 +15,14 @@ Gem::Specification.new do |s|
   s.executables      = s.files.grep(%r{^bin/}) { |file| File.basename(file) }
   s.homepage         = 'https://github.com/theforeman/smart_proxy_salt'
   s.license          = 'GPL-3.0'
+
+  s.add_development_dependency 'bundler', '~> 1.7'
```

Comment:
The inconsistency with parenthesis irks me

Reply:
so true. Thanks :)